[Use Cases] Relying Parties, PID Provider, Attestation Provider and Wallet Provider Onboarding

ChiaLeon opened the pull request:
Proposing a draft of "PID / Attestation Provider Onboarding" related to issue webuild-consortium/wp4-architecture#5

RosaliaGaleano committed:
Wallet-Relying Party Onboarding

Proposing a draft of "Wallet-Relying Party Onboarding"

RosaliaGaleano committed:
Update Wallet-Relying Party Onboarding

Editorial fix
Update requirement bullet points

RosaliaGaleano committed:
Update task1-use-cases/subtask1-1-onboarding/Wallet-Relying Party Onboarding

Co-authored-by: Giuseppe De Marco <demarcog83@gmail.com>

RosaliaGaleano committed:
Apply suggestions from code review

Co-authored-by: Giuseppe De Marco <demarcog83@gmail.com>

ChiaLeon committed:
Create PID & Attestation Provider Onboarding

Proposing a draft of "PID / Attestation Provider Onboarding" related to issue #5

RosaliaGaleano committed:
Update and rename Wallet-Relying Party Onboarding to Relying Party Onboarding

New and more detailed content.

sander commented on task1-use-cases/subtask1-1-onboarding/PID & Attestation Provider Onboarding:
Please rename with a `.md` suffix so that GitHub properly renders the Markdown.

peppelinux replied:
just did it

sander commented on task1-use-cases/subtask1-1-onboarding/PID & Attestation Provider Onboarding:
```diff
@@ -0,0 +1,570 @@
+        - indication whether the wallet-relying party is a public sector body
+        - applicable entitlement(s) of the wallet-relying party chosed between:
+            - Service_Provider
+            - **QEAA_Provider**
```
We should align on criteria for acceptance as QEAA Provider within WE BUILD.

sander reviewed (commented):
Thank you @ChiaLeon for publishing this draft and @peppelinux for inviting me to review.

peppelinux committed:
fix: rename file using markdow file extension

peppelinux committed:
fix: rename file using markdow file extension

peppelinux committed:
fix: rename file using markdow file extension

alejandroalffer commented:
Thank you for this contribution, @ChiaLeon !

The structure and the onboarding flow are very clear and helpful.

We would like to propose two technical enhancements for future iterations of the onboarding specification:

1. **Early Binding via Electronic Employee Certificates**
Incorporating support for employee or representative electronic certificates (e.g., QSeal/QSign corporate certificates) would enable strong early binding between the legal entity and the individual acting on its behalf. This not only provides cryptographic proof of organisational affiliation during onboarding, but can also be complemented with an electronically signed role-acceptance statement or domain-validated evidence, ensuring explicit confirmation of the individual’s mandate. This approach aligns with ETSI EN 319 412-3 and established corporate identity-binding practices.

2. **Integration with Authoritative Sources (“Sources of Truth”)**
Adding optional checks against authoritative registries such as GLEIF (LEI records) would enable enriched, verified organisational metadata, improve trust scoring, and support automated consistency checks between submitted certificates, policy data, and official entity information.

These two capabilities would strengthen identity assurance, automate validation steps, and improve interoperability with eIDAS2 and wider trust frameworks.

Thanks again for the excellent work!

Looking forward to continuing the discussion! :-)

DavSanOl commented on task1-use-cases/subtask1-1-onboarding/pid_eaa_provider_onboarding.md:
```diff
@@ -0,0 +1,570 @@
+    - When a Registrar suspends or cancels registration of a PID Provider or Attestation Provider, the **PID Provider or Attestation Provider revokes all of their PIDs or attestations** as described in Section 6.6.3.7.
+
+==- The Access Certificate Authority publishes the Access Certificate revocation status (within 24 hours)==
+==- Provider of Registration Certificate publishes the Access Certificate revocation status (within 24 hours)==
```
Great work @ChiaLeon!
I'd just like to highlight these two lines: the Provider of Registration Certificate publishes the Registration Certificate revocation status, not the Access Certificate revocation status, right?

peppelinux committed:
Apply suggestions from code review

Co-authored-by: grimilde <gabriella.cefalu@gmail.com>

MartinRobomaze committed:
initial draft of wallet onboarding process

cklugow committed:
Update task1-use-cases/subtask1-3-wallet-provider-onboarding/README.md

Co-authored-by: Lukasz Jaromin <lukasz.jaromin@raidiam.com>

cklugow committed:
Updated Title in Line 6

MartinRobomaze committed:
fixed typos etc

IDUhammer committed:
Update README.md

IDUhammer committed:
Update README.md

cklugow committed:
date changed

cklugow committed:
MVP/MVP+ labeling an changing the order

cklugow committed:
RACI Matrix filled - first version

cklugow committed:
added link to Interoperability Test Bed and responsibility of WP4 Testing

cklugow committed:
erased obsolete content (see line 84)

cklugow committed:
entering an assumption for the service unique identifier of the wallet solution

cklugow committed:
added some references to Regulation (EU) No. 2024/1183 5d

cklugow committed:
changing format

cklugow committed:
changes in format

cklugow committed:
Merge branch 'main' into wallet-provider-onboarding

peppelinux committed:
Merge branch 'main' into wallet-provider-onboarding

peppelinux committed:
fix: removed duplication in wallet provider merge

peppelinux committed:
Apply suggestions from code review

Co-authored-by: Marco Pernpruner <marco.pernpruner@gmail.com>

peppelinux committed:
editorial: fix rp onboarding terms

peppelinux committed:
Merge branch 'patch-1' of https://github.com/RosaliaGaleano/wp4-trust-group into patch-1

peppelinux committed:
editorial: fix rp onboarding duplications and ARF alignments

peppelinux committed:
editorial: fix rp onboarding duplications and ARF alignments

peppelinux committed:
Merge branch 'main' into patch-1

peppelinux committed:
editorial: fix rp onboarding duplications and ARF alignments

peppelinux committed:
editorial: fix rp onboarding

peppelinux committed:
editorial: fix rp onboarding with @grimilde reviews

peppelinux committed:
Merge pull request #7 from RosaliaGaleano/patch-1

Relying Party Onboarding

peppelinux committed:
editorial: pid eaa provider onboarding

peppelinux committed:
feat: add onboarding base framework document

- Create onboarding-base.md with common onboarding framework
- Document common success criteria, preconditions, and main flow structure
- Add API schema alignment note
- Prepare for entity-specific onboarding documents to reference this base

peppelinux committed:
feat: complete onboarding framework with MVP/MVP+ structure and schema harmonization

- Complete onboarding-base.md with all common sections (preconditions, main flow, industrial-scale considerations, administrative/technical/post-onboarding steps, normative references)
- Update relying_party_onboarding.md with MVP/MVP+ structure and base references
- Update pid_eaa_provider_onboarding.md with MVP/MVP+ structure and base references
- Add schema harmonization notes to onboarding-api/README.md and task5 README.md
- Harmonize data models across all specifications (participant types, statuses, certificate types)
- Update Registration Workflow in onboarding-api to reference detailed onboarding phases
- Apply WEBUILD MVP (testing/pilot) and MVP+ (production/regulatory) distinctions throughout

cklugow committed:
Update README.md

Final check before merging
- no dependency to conformity tests in webuild
- no WUA specification
- editorial changes

cklugow committed:
Merge pull request #43 from webuild-consortium/wallet-provider-onboarding

Wallet Provider Onboarding

peppelinux committed:
Merge branch 'main' into ChiaLeon-patch-1

peppelinux committed:
chore: onboarding use cases harmonization

andrea-dintino committed:
Add Forkbomb BV to onboarding entities list

peppelinux committed:
Apply suggestions from code review

peppelinux committed:
editorial: onboarding use case restructure and improvements

andmoro reviewed (commented):
Some minor comments here and there but solid PR overall.

peppelinux committed:
chore: clarification about LoTL Provider

peppelinux committed:
editorial: refs to ETSI Registration Certificates and removal of exceeding IETF refs

peppelinux committed:
fix: Clarified that the Commission is notified about the Registrar and the National Register (Article 3) and does not maintain a register of Relying Parties.

peppelinux committed:
editorial: clarification about who updates the TLs

nklomp reviewed (commented):
Only have editorial suggestions, and mainly a question about natural person as PID provider.

peppelinux committed:
editorial: onboarding, removed duplicated terms

peppelinux committed:
fix: removed PID provider natiural person

peppelinux committed:
chore: clarification about LoTL and TLs in onboarding

peppelinux committed:
Apply suggestion from @nklomp

Co-authored-by: Niels Klomp <nklomp@users.noreply.github.com>

peppelinux committed:
chore: clarification about LoTL and TLs in onboarding

peppelinux committed:
Merge branch 'ChiaLeon-patch-1' of https://github.com/webuild-consortium/wp4-trust-group into ChiaLeon-patch-1

nklomp reviewed (approved):
Only remaining question is in the RACI whether we want to add a line for testing the issuer by the test group, or whether we defer that to a ticket so we can add it later @peppelinux 

Rest LGTM, so approved
